sequencer/interfaces.go

```diff
@@ -96,4 +99,6 @@ type dbManagerStateInterface interface {
 	GetLastL2Block(ctx context.Context, dbTx pgx.Tx) (*types.Block, error)
 	MarkReorgedTxsAsPending(ctx context.Context) error
 	GetLatestGer(ctx context.Context) (state.GlobalExitRoot, time.Time, error)
+	GetLatestGlobalExitRoot(ctx context.Context, maxBlockNumber uint64, dbTx pgx.Tx) (state.GlobalExitRoot, time.Time, error)
+	GetSender(tx types.Transaction) (common.Address, error)
```

Comment:
this `GetSender` function is not on the state struct but is just defined as a standalone function: https://github.com/0xPolygonHermez/zkevm-node/blob/develop/state/transaction.go#L9